lsp/nls/src/linearization/completed.rs

```diff
@@ -37,9 +37,8 @@ impl Completed {
         LinearizationItem { env, .. }: &LinearizationItem<Resolved>,
     ) -> Vec<&LinearizationItem<Resolved>> {
         env.iter()
-            .map(|(_, id)| self.get_item(*id))
-            .collect::<Option<Vec<_>>>()
-            .unwrap_or_default()
+            .filter_map(|(_, id)| self.get_item(*id))
+            .collect::<Vec<_>>()
```

Comment:
Tiny nitpick: you'll probably find you don't need the `::<Vec<_>>` turbofish anymore. Since we're not calling `unwrap_or_default` after `collect`, the function's return type should be enough to infer what the type of `collect` has to be here.

Reply:
Yeah, thank!